client/src/extension.ts

```diff
@@ -184,7 +184,7 @@ function handleConfigurationChange(event: vscode.ConfigurationChangeEvent) {
       event.affectsConfiguration("deno.path") ||
       event.affectsConfiguration("deno.maxTsServerMemory")
     ) {
-      vscode.commands.executeCommand("deno.restart");
+      vscode.commands.executeCommand("deno.client.restart");
```

Comment:
This is not gonna be backwards compatible; can we make it backwards compatible for 1-3 months?

Reply:
Err, this is only `vscode_deno`. Doesn't have to do anything with the LSP.